pkg/cache/cache.go

```diff
@@ -144,9 +144,13 @@ type Options struct {
 	// instead of `reconcile.Result{}`.
 	SyncPeriod *time.Duration
 
-	// Namespaces restricts the cache's ListWatch to the desired namespaces
-	// Per default ListWatch watches all namespaces
-	Namespaces []string
+	// DefaultNamespaces maps namespace names to cache settings. If set, only
```

Comment:
```suggestion
	// DefaultNamespaces maps namespace names to cache configs. If set, only
```
nit

(same for other occurences of "cache settings" below)

Reply:
Done